servo-tidy.toml

```diff
@@ -82,7 +82,6 @@ packages = [
     "uuid",
 
     # winit port minibrowser (servo/servo#30049)
```

Comment:
this comment can now be removed. the ignore for ahash was actually added in #29795, and unrelated to the minibrowser.

Reply:
Thanks for the review, @delan. I've pushed a new version of this change the removes the comment and moves `ahash` into the main body of the exclusions.